client/apps/edit/components/content2/sections/header/index.styl

```diff
@@ -0,0 +1,81 @@
+.edit-header
+  padding-top 60px
+  .public-DraftEditorPlaceholder-root
```

Comment:
Is it possible to use an internal class then this rather than the third party's? It's less predictable if their underlying code changes. If not, no worries.